config/pmd/ruleset.xml

```diff
@@ -41,6 +41,7 @@
     <exclude name="TooManyStaticImports" />
     <exclude name="UseExplicitTypes" />
     <exclude name="UseUnderscoresInNumericLiterals" />
+    <exclude name="TypeParameterNamingConventions" />
```

Comment:
Although this is a new rule, it doesn't fit our usage. For example, we use *word* for parameter names but this rule enforces to use a single letter.

https://github.com/apache/spark-kubernetes-operator/blob/c07e61eaf17b785777b49a078efe085f18164d3a/spark-operator-api/src/main/java/org/apache/spark/k8s/operator/BaseResource.java#L34-L45